resources/META-INF/plugin.xml.template

```diff
@@ -8,20 +8,35 @@
 
   <category>Custom Languages</category>
 
-  <version>18.3</version>
+  <version>18.4</version>
 
   <idea-version since-build="@SINCE@" until-build="@UNTIL@"/>
 
+  <depends>Dart</depends>
+
   <!-- plugin compatibility -->
   <!-- see: http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html -->
-  @DEPENDS@
```

Comment:
Won't this cause 'Flutter' to show up twice in the plugin manager?

Reply:
This is shipping support for both under the `io.flutter` id; I think we'll just see this once total. Totally happy to re-evaluate the plugin distro plan once we have this release under our belts :)